client_vpn/input.tf

```
@@ -120,3 +107,21 @@ variable "vpc_id" {
   description = "(Required) ID of the VPC to associate with the VPN endpoint."
   type        = string
 }
+variable "authentication_option" {
+  description = "(Optional, default 'federated-authentication') The authentication option to use for the VPN endpoint.  Valid values are 'federated-authentication' or 'certificate-authentication'."
+  type        = string
+  default     = "federated-authentication"
+}
```

Just as a nice to have, could you add validation:
```suggestion
  validation {
    condition     = contains(["federated-authentication", "certificate-authentication"], var.authentication_option)
    error_message = "The authentication option must be either 'federated-authentication' or 'certificate-authentication'."
  } 
}

```